runtime/common/src/lib.rs

```diff
@@ -14,6 +14,9 @@ pub type Signature = MultiSignature;
 /// to the public key of our transaction signing scheme.
 pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
 
+/// Type for IDs of farming pools
+pub type PoolId = u32; //pool id for farming rpc api
```

Comment:
Nit: The other comment is probably not necessary anymore.
```suggestion
pub type PoolId = u32;
```